src/components/members-list/index.js

```diff
@@ -3,18 +3,31 @@ import MemberListItem from '@components/member-list-item';
 import classNames from '@components/members-list/member-list.module.scss';
 import { membersContext } from '@store/members/members-context';
 
-const MembersList = () => {
+const MembersList = ({ searchTerm }) => {
   const {
     state: { membersArr },
   } = membersContext();
   if (membersArr) {
     return (
       <div className={classNames.container}>
-        {membersArr.map((ele) => (
-          <React.Fragment key={ele.id}>
-            <MemberListItem memberDetails={ele} />
-          </React.Fragment>
-        ))}
+        {membersArr.length > 0 &&
+          membersArr
+            .filter(
+              (ele) =>
+                ele.first_name
+                  .toLowerCase()
+                  .includes(searchTerm.toLocaleLowerCase()) ||
+                ele.last_name
+                  .toLowerCase()
+                  .includes(searchTerm.toLocaleLowerCase())
```

Comment:
```js

  (ele) =>
                ele.first_name
                  .toLowerCase()
                  .includes(searchTerm.toLocaleLowerCase()) ||
                ele.last_name
                  .toLowerCase()
                  .includes(searchTerm.toLocaleLowerCase())

```
The above code is repeated in `members` and `new members` for now and it goes for different categories.
Why don't we think of having a separate function, which can be added in `utils/` and we can import everywhere?
Let me know you view

Reply:
Yes, we can write a separate function for that. But 
1. To write a separate function for just one filter condition is not suitable I think.
`(ele.first_name.toLowerCase().includes(searchTerm.toLocaleLowerCase()) ||(ele.last_name.toLowerCase().includes(searchTerm.toLocaleLowerCase()) `
2. Neither the lines of code added are much more nor it is being used at more places.
So that's why I prefered not to go with `utils/` and modified the existing code in a better way.

Reply:
agree with @harshith-venkatesh, please create a util function, and let's move it there.

Reply:
done